### DISCUSSION TITLE
Introduction and Collaboration

### DISCUSSION THREAD

**Discussion by VahidGh** [openworm/worm-math-book#6] [Show and tell]
Please introduce yourself over here and let us know about your interest areas.

**Comment by manuel-delverme** [comment]
Hello!
I'm trying to understand what kind of realistic biological simulators are available right now and OpenWorm is something I've been following for a long time.

I'm a PhD student at https://mila.quebec/ working on control and reinforcement learning.
My interest is in dynamical systems, and continuous time models.

With a colleague we are looking at how we can create parametric models (neural networks) of life, starting from the genome (genome -> structure -> behavior), this is obviously a moonshot but it seems your project covers all the layers from structure to behavior which makes it quite interesting.

**Reply by VahidGh** [reply]
Hey @manuel-delverme, welcome!

Thanks for your comment, and sorry for my late response.
Actually, I'm planning this work out these days (preferably, under supervision of some professor and collaboration of more contributors).

The first issue that we should come up with before going further is this:
https://github.com/openworm/worm-math-book/discussions/4
So, please let me know what would be your preference.

You might also fill out [this form](https://goo.gl/3ncZWn) for an invitation to our Slack channel for further conversations.
